lib/archethic/transaction_chain/transaction_summary.ex

```diff
@@ -207,4 +212,55 @@ defmodule Archethic.TransactionChain.TransactionSummary do
       validation_stamp_checksum: validation_stamp_checksum
     }
   end
+
+  @doc """
+  This function will be used during the summary day of 1.0.8 upgrade. This function can be deleted after the upgrade.
+  Migrate this function into files 1.0.8-migrate_old_tx_summaries
+  Transform a TransactionSummary without version into a TransactionSummary v1
+  """
+  @spec transform_1_0_8_summary(t()) :: t()
+  def transform_1_0_8_summary(tx_summary = %__MODULE__{version: 1}), do: tx_summary
```

Comment:
Why is it in this file ? I think the migration should have this function, hence do no need to alter this file after the upgrade, so the function will be contextualized for the migration only.

Reply:
I think it would be clearer to use a `Slot.transform/2` and `TransactionSummary.transform/2` which just do transformation of data structure.